src/python/pathops/_pathops.pyx:691:6: Variables cannot be declared with 'cpdef'. Use 'cdef' instead.

diff --git a/src/python/pathops/_pathops.pxd b/src/python/pathops/_pathops.pxd
index 0b01fa9..6e12db0 100644
--- a/src/python/pathops/_pathops.pxd
+++ b/src/python/pathops/_pathops.pxd
@@ -213,5 +213,5 @@ cdef uint8_t *POINTS_IN_VERB
 
-cpdef dict VERB_METHODS
+cdef dict VERB_METHODS
 
-cpdef dict PEN_METHODS
+cdef dict PEN_METHODS
 
diff --git a/src/python/pathops/_pathops.pyx b/src/python/pathops/_pathops.pyx
index 7f1517f..7a3d557 100644
--- a/src/python/pathops/_pathops.pyx
+++ b/src/python/pathops/_pathops.pyx
@@ -690,3 +690,3 @@ cdef uint8_t *POINTS_IN_VERB = [
 
-cpdef dict VERB_METHODS = {
+cdef dict VERB_METHODS = {
     kMove_Verb: "moveTo",
@@ -699,3 +699,3 @@ cpdef dict VERB_METHODS = {
 
-cpdef dict PEN_METHODS = {
+cdef dict PEN_METHODS = {
     kMove_Verb: "moveTo",
